sirepo/pkcli/roles.py

```diff
@@ -46,13 +46,30 @@ def delete_roles(*args):
     delete(*args)
 
 
+def disable_user(moderator_uid, uid_or_email):
+    """Remove role user
+    Args:
```

Comment:
Formatting isn't right. Needs to be indented.

Reply:
[609c03c](https://github.com/radiasoft/sirepo/pull/7070/commits/609c03c32717dcd27067fe26bc7c2bbb4ee16dc1)